### PR TITLE
add ability to resize packer

### LINKED ISSUE #1: Allow packer to be resized
Title is pretty self-explanatory, I can work on a PR if you like.

### PR DESCRIPTION
**Pull request by ashkitten**
This resolves #1

### CODE REVIEW

File: src/packer/packer.rs [hunk 1]
```diff
@@ -51,6 +51,26 @@ impl DensePacker {
...
+        assert!(width >= self.width && height >= self.height);
+
+        self.width = width;
+        self.height = height;
+
+        // Add a new skyline to fill the gap
+        // The new skyline starts where the furthest one ends
+        let mut left = 0;
+        for skyline in self.skylines.iter() {
+            left = max(left, skyline.left);
```
Comment (kryptan) [inline src/packer/packer.rs]: I don't remember how all this stuff works exactly but shouldn't it be `left = max(left, skyline.right());` instead?

Also, is the loop necessary? Skylines are probably already sorted and you can just take the last one if I remember correctly.

**Comment by kryptan** [comment]
Thanks, didn't know someone is using this library)

Have you tested your changes?

File: src/packer/packer.rs (hunk 1 above)
Comment (ashkitten) [reply src/packer/packer.rs]: You are correct, I have amended those changes.

**Comment by ashkitten** [comment]
I have tested with [my application](https://gitlab.com/ashkitten/yotredash/tree/vulkano). There was a slight bug before, but with the changes that you requested it's gone now. I tested at font sizes 100, 150, 200, 300, 500, and 1000 and the glyph atlas seems to come out right every time.

**Comment by ashkitten** [comment]
bump

**Pull request merged**